keyboards/viktus/sp111_v2/config.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 BlindAssassin111 (@blindassassin111)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* Split Defines */
+#define SPLIT_USB_DETECT
```

Comment:
Which is it?
https://github.com/thompson-ele/qmk_firmware/blob/38cae8d3adf9ee1c6566a05c666b26d9afbda71f/keyboards/viktus/sp111_v2/info.json#L65-L67
Either way, it should be removed from `config.h` file.
```suggestion
```